site/src/pages/RoadmapPage/Planner.tsx

```diff
@@ -162,6 +166,7 @@ const Planner: FC = () => {
             // prerequisite not fulfilled, has some required classes to take
             if (required.size > 0) {
               console.log('invalid course', course.id);
+              // console.log('missing', course.prerequisite_text);
```

Comment:
Great for debugging but I think you can go ahead and delete this if you're not using it